packages/react-components/react-tabster/src/focus/createCustomFocusIndicatorStyle.ts

```diff
@@ -19,5 +19,17 @@ export const createCustomFocusIndicatorStyle = (
   ':focus': {
     outlineStyle: 'none',
   },
-  [`${KEYBOARD_NAV_SELECTOR} :${selector}`]: style,
+  ':focus-visible': {
+    outlineStyle: 'none',
+  },
+  ...(selector === 'focus' && {
+    [`:global(.fui-FluentProvider)`]: {
+      [`& .${FOCUS_VISIBLE_CLASS}`]: style,
+    },
+  }),
+  ...(selector === 'focus-within' && {
+    [`:global(.fui-FluentProvider)`]: {
+      [`& .${FOCUS_WITHIN_CLASS}:${selector}`]: style,
+    },
+  }),
```

Comment:
Can you please clarify why we need this for `:focus-within` if it's natively supported by [our browser-matrix](https://github.com/microsoft/fluentui/blob/master/rfcs/shared/build-system/06-browser-support-for-v9.md#detailed-design-or-proposal)?

![image](https://user-images.githubusercontent.com/14183168/182351771-64d5ecce-68b2-4b10-8238-824c758b10f0.png)
![image](https://user-images.githubusercontent.com/14183168/182351911-a62779b1-90c3-4be9-b323-6c4d7c9e5a3e.png)

Reply:
yes because `focus-within` does not support keyboard/mouse heuristics https://github.com/WICG/focus-visible/issues/151

I can add documentation to that effect

Reply:
Yes, please add a comment as it might confuse people why have code around the natively supported feature.